packages/route-pattern/src/lib2/route-pattern/parse.test.ts

```diff
@@ -0,0 +1,68 @@
+import { describe, expect, it } from 'vitest'
```

Comment:
Any reason we can't use `node:test` here? I'd prefer not to use two different test tools if we can avoid it.